controllers/data.controller.js

```diff
@@ -43,669 +60,613 @@ const client = new MongoClient(uri);
-                    updatedTier = "E4";
-                    updatedScore = 0;
-                } else if (currentTier === "E4" && currentScore >= 100) {
-                    updatedTier = "E3";
-                    updatedScore = currentScore - 100;
-                // 에메랄드3
-                } else if (currentTier === "E3" && currentScore < 0) {
-                    updatedTier = "E4";
-                    updatedScore = currentScore + 100;
-                } else if (currentTier === "E3" && currentScore >= 100) {
-                    updatedTier = "E2";
-                    updatedScore = currentScore - 100;
-                // 에메랄드2
-                } else if (currentTier === "E2" && currentScore < 0) {
-                    updatedTier = "E3";
-                    updatedScore = currentScore + 100;
-                } else if (currentTier === "E2" && currentScore >= 100) {
-                    updatedTier = "E1";
-                    updatedScore = currentScore - 100;
-                // 에메랄드1
-                } else if (currentTier === "E1" && currentScore < 0) {
-                    updatedTier = "E2";
-                    updatedScore = currentScore + 100;
-                } else if (currentTier === "E1" && currentScore >= 100) {
-                    updatedTier = "D4";
-                    updatedScore = currentScore - 100;
-                
-                // [다이아몬드] 티어
-                // 다이아몬드4
-                } else if (currentTier === "D4" && currentScore < 0) {
-                    updatedTier = "D4";
-                    updatedScore = 0;
-                } else if (currentTier === "D4" && currentScore >= 100) {
-                    updatedTier = "D3";
-                    updatedScore = currentScore - 100;
-                // 다이아몬드3
-                } else if (currentTier === "D3" && currentScore < 0) {
-                    updatedTier = "D4";
-                    updatedScore = currentScore + 100;
-                } else if (currentTier === "D3" && currentScore >= 100) {
-                    updatedTier = "D2";
-                    updatedScore = currentScore - 100;
-                // 다이아몬드2
-                } else if (currentTier === "D2" && currentScore < 0) {
-                    updatedTier = "D3";
-                    updatedScore = currentScore + 100;
-                } else if (currentTier === "D2" && currentScore >= 100) {
-                    updatedTier = "D1";
-                    updatedScore = currentScore - 100;
-                // 다이아몬드1
-                } else if (currentTier === "D1" && currentScore < 0) {
-                    updatedTier = "D2";
-                    updatedScore = currentScore + 100;
-                } else if (currentTier === "D1" && currentScore >= 100) {
-                    updatedTier = "M";
-                    updatedScore = currentScore - 100;
-                } 
-
-            }
-
+  try {
+    await client.connect();
+
+    const db = client.db("test");
```

Comment:
I think accessing to db and coll in every function is not efficient. Maybe you can create new model or service so you can use it in every function